tests/test_server_proto.py

```diff
@@ -2212,6 +2212,10 @@ async def test_server_adjacent_database_propagation(self):
                     await con2.query("select 1")
                     await con2.aclose()
 
+            await self.con.execute('''
+                DROP DATABASE test_db_prop;
+            ''')
```

Comment:
Should we test *recreating* a database with the same name, after a drop?

Reply:
(And connecting to the new one, I guess)

Reply:
Done.